.github/workflows/sonarcloud.yml

```diff
@@ -0,0 +1,56 @@
+name: SonarCloud
+
+on:
+  push:
+    branches:
+      - dev
+
+jobs:
+  static_analysis:
+    runs-on: ubuntu-latest
+
+    container:
+        image: navitia/debian10_dev
+
+    services:
+      rabbitmq:
+        image: rabbitmq:3-alpine
```

Comment:
I don't know, why you use rabbit. 
I think **rabbit** is only use with docker test (_make docker_test_) ?
I don't know if I am right.

Reply:
You're right, my bad, I'll remove it !